base/ltexpl.dtx

```diff
@@ -516,4 +516,27 @@
 %<latexrelease>\EndIncludeInRelease
 %    \end{macrocode}
 %
+% \DescribeMacro\IfExplAtLeastTF
+%   A pretty simple wrapper.
+%
+% \begin{macro}{\IfExplAtLeastTF}
+% \changes{v1.3g}{2023/10/13}{Provide a test for \pkg{expl3} date (gh/1004)}
+%    \begin{macrocode}
+%<*2ekernel|latexrelease>
+%<latexrelease>\IncludeInRelease{2023/11/01}%
+%<latexrelease>                 {\IfExplAtLeastTF}{Test for expl3 date}%
+\def\IfExplAtLeastTF{\@ifl@t@r\ExplLoaderFileDate}
```

Comment:
not sure, but to address the comment of \ExplLoderFileDate not available in case of a roll forward on could have an additional line here that sets it to something (not sure what) if it is undefined.

However, my guess is that such a roll forward would not work anyway, so probably not really helpful

Reply:
`\ExplLoaderFileDate` has been defined since 2017 (https://github.com/latex3/latex3/commit/d060bf686b5982ef5ee20927369a70fc17df0240#diff-f1498259ee1774283bc5d8fea536a113cac11edca429dddab2695291d24a0444) - I think if it doesn't exist there are other issues to worry about!

Reply:
Then I think we are good if it is that old.